{"uid":"8ebed90aba785007","name":"QR type - webkit","fullName":"tests.tests_default_flow.test_default_sign_up_eps_download_welcome_modal.TestDefaultSignUpFlow#test_default_sign_up_qr_type","historyId":"8fb28a6801bf1c1e523f6e8e9a15ca2c","time":{"start":1733423735526,"stop":1733423815464,"duration":79938},"status":"broken","statusMessage":"playwright._impl._errors.TimeoutError: Locator.fill: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//input[@id='vcard_phoneLabel']\")","statusTrace":"self = <tests.tests_default_flow.test_default_sign_up_eps_download_welcome_modal.TestDefaultSignUpFlow object at 0x7f52a2379610>\nsign_up_fixture = None, qr_create_method = 'vcard_qr_create'\nfake_email = 'wtl-automation2614762965835@test.com', resolution = 'Default'\n\n    @allure.title(f\"QR type - {os.getenv('BROWSER')}\")\n    @pytest.mark.parametrize(\"qr_create_method\", qr_create_methods)\n    @pytest.mark.parametrize(\"resolution\", resolution_qr_code_images)\n    def test_default_sign_up_qr_type(self, sign_up_fixture, qr_create_method, fake_email, resolution):\n        qr_create_method_func = getattr(self.qr_creation_page, qr_create_method)\n>       qr_create_method_func()\n\ntests/tests_default_flow/test_default_sign_up_eps_download_welcome_modal.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/qr_creation_flow_page.py:104: in vcard_qr_create\n    self.helper.add_phone_email_website()\npages/qr_code_helper.py:176: in add_phone_email_website\n    self.locator.contact_details_qr_code_add_phone_label.fill(self.faker.word())\n/usr/local/lib/python3.12/dist-packages/playwright/sync_api/_generated.py:15743: in fill\n    self._sync(\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_locator.py:208: in fill\n    return await self._frame.fill(self._selector, strict=True, **params)\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_frame.py:527: in fill\n    await self._channel.send(\"fill\", locals_to_params(locals()))\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7f52a293c5c0>\ncb = <function Channel.send.<locals>.<lambda> at 0x7f52a0aa5620>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.fill: Timeout 30000ms exceeded.\nE           Call log:\nE           waiting for locator(\"//input[@id='vcard_phoneLabel']\")\n\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py:520: TimeoutError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1733423422846,"stop":1733423422846,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1733423422846,"stop":1733423422846,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_session_faker","time":{"start":1733423422847,"stop":1733423422947,"duration":100},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"pytestconfig","time":{"start":1733423422847,"stop":1733423422847,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"delete_output_dir","time":{"start":1733423422847,"stop":1733423422847,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Set up browser: webkit","time":{"start":1733423422947,"stop":1733423425408,"duration":2461},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Failed test artifacts","time":{"start":1733423731517,"stop":1733423731517,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"fake_email","time":{"start":1733423731517,"stop":1733423731517,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Delete user after test","time":{"start":1733423731517,"stop":1733423731517,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"context","time":{"start":1733423731517,"stop":1733423731604,"duration":87},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"page","time":{"start":1733423731605,"stop":1733423732086,"duration":481},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Sign up","time":{"start":1733423732086,"stop":1733423735525,"duration":3439},"status":"passed","steps":[{"name":"Sign up","time":{"start":1733423734436,"stop":1733423735525,"duration":1089},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"temporary_mail","value":"'wtl-automation2614762965835@test.com'"},{"name":"signup_password","value":"'wtl-automation2614762965835@test.com'"}],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},{"name":"setup","time":{"start":1733423732086,"stop":1733423732086,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"playwright._impl._errors.TimeoutError: Locator.fill: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//input[@id='vcard_phoneLabel']\")","statusTrace":"self = <tests.tests_default_flow.test_default_sign_up_eps_download_welcome_modal.TestDefaultSignUpFlow object at 0x7f52a2379610>\nsign_up_fixture = None, qr_create_method = 'vcard_qr_create'\nfake_email = 'wtl-automation2614762965835@test.com', resolution = 'Default'\n\n    @allure.title(f\"QR type - {os.getenv('BROWSER')}\")\n    @pytest.mark.parametrize(\"qr_create_method\", qr_create_methods)\n    @pytest.mark.parametrize(\"resolution\", resolution_qr_code_images)\n    def test_default_sign_up_qr_type(self, sign_up_fixture, qr_create_method, fake_email, resolution):\n        qr_create_method_func = getattr(self.qr_creation_page, qr_create_method)\n>       qr_create_method_func()\n\ntests/tests_default_flow/test_default_sign_up_eps_download_welcome_modal.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/qr_creation_flow_page.py:104: in vcard_qr_create\n    self.helper.add_phone_email_website()\npages/qr_code_helper.py:176: in add_phone_email_website\n    self.locator.contact_details_qr_code_add_phone_label.fill(self.faker.word())\n/usr/local/lib/python3.12/dist-packages/playwright/sync_api/_generated.py:15743: in fill\n    self._sync(\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_locator.py:208: in fill\n    return await self._frame.fill(self._selector, strict=True, **params)\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_frame.py:527: in fill\n    await self._channel.send(\"fill\", locals_to_params(locals()))\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7f52a293c5c0>\ncb = <function Channel.send.<locals>.<lambda> at 0x7f52a0aa5620>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.fill: Timeout 30000ms exceeded.\nE           Call log:\nE           waiting for locator(\"//input[@id='vcard_phoneLabel']\")\n\n/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py:520: TimeoutError","steps":[{"name":"Create VCard QR code","time":{"start":1733423735704,"stop":1733423815464,"duration":79760},"status":"broken","statusMessage":"playwright._impl._errors.TimeoutError: Locator.fill: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//input[@id='vcard_phoneLabel']\")\n\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/app/workspace/pages/qr_creation_flow_page.py\", line 104, in vcard_qr_create\n    self.helper.add_phone_email_website()\n  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/app/workspace/pages/qr_code_helper.py\", line 176, in add_phone_email_website\n    self.locator.contact_details_qr_code_add_phone_label.fill(self.faker.word())\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/sync_api/_generated.py\", line 15743, in fill\n    self._sync(\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_locator.py\", line 208, in fill\n    return await self._frame.fill(self._selector, strict=True, **params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_frame.py\", line 527, in fill\n    await self._channel.send(\"fill\", locals_to_params(locals()))\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py\", line 59, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py\", line 520, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n","steps":[{"name":"Close help modal window step 2","time":{"start":1733423746141,"stop":1733423752390,"duration":6249},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Add phone, email and website","time":{"start":1733423769283,"stop":1733423815463,"duration":46180},"status":"broken","statusMessage":"playwright._impl._errors.TimeoutError: Locator.fill: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//input[@id='vcard_phoneLabel']\")\n\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/app/workspace/pages/qr_code_helper.py\", line 176, in add_phone_email_website\n    self.locator.contact_details_qr_code_add_phone_label.fill(self.faker.word())\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/sync_api/_generated.py\", line 15743, in fill\n    self._sync(\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_locator.py\", line 208, in fill\n    return await self._frame.fill(self._selector, strict=True, **params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_frame.py\", line 527, in fill\n    await self._channel.send(\"fill\", locals_to_params(locals()))\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py\", line 59, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/playwright/_impl/_connection.py\", line 520, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":2,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":3,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"Sign up::0","time":{"start":1733423815585,"stop":1733423815585,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"page::0","time":{"start":1733423815586,"stop":1733423817727,"duration":2141},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Delete user after test::0","time":{"start":1733423817728,"stop":1733423817902,"duration":174},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"context::0","time":{"start":1733423817728,"stop":1733423817728,"duration":0},"status":"broken","statusMessage":"TypeError: Tracing.stop() got an unexpected keyword argument 'name'\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/app/workspace/conftest.py\", line 27, in context\n    context.tracing.stop(name=f\"artifacts/traces/{request.node.name}.zip\")\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Failed test artifacts::0","time":{"start":1733423817903,"stop":1733423817907,"duration":4},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/traces/test_default_sign_up_qr_type[Default-vcard_qr_create-en].zip'\n","statusTrace":"  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/app/workspace/conftest.py\", line 57, in artifacts\n    allure.attach.file(f\"artifacts/traces/{request.node.name}.zip\", name=\"trace\",\n  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/_allure.py\", line 203, in file\n    plugin_manager.hook.attach_file(source=source, name=name, attachment_type=attachment_type, extension=extension)\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_callers.py\", line 139, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/allure_pytest/listener.py\", line 256, in attach_file\n    self.allure_logger.attach_file(uuid4(), source, name=name, attachment_type=attachment_type, extension=extension)\n  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/reporter.py\", line 160, in attach_file\n    plugin_manager.hook.report_attached_file(source=source, file_name=file_name)\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_callers.py\", line 139, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"/usr/local/lib/python3.12/dist-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/dist-packages/allure_commons/logger.py\", line 39, in report_attached_file\n    shutil.copy2(source, destination)\n  File \"/usr/lib/python3.12/shutil.py\", line 475, in copy2\n    copyfile(src, dst, follow_symlinks=follow_symlinks)\n  File \"/usr/lib/python3.12/shutil.py\", line 260, in copyfile\n    with open(src, 'rb') as fsrc:\n         ^^^^^^^^^^^^^^^\n","steps":[],"attachments":[{"uid":"9f6bb9d03ce3ade1","name":"screenshot","source":"9f6bb9d03ce3ade1.png","type":"image/png","size":253742},{"uid":"9eac31be6ee7f9ed","name":"video","source":"9eac31be6ee7f9ed.webm","type":"video/webm","size":2454803},{"name":"trace","type":"application/zip","size":0}],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Set up browser: webkit::0","time":{"start":1733423817908,"stop":1733423818000,"duration":92},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Default sign up flow - webkit - webkit"},{"name":"parentSuite","value":"tests.tests_default_flow"},{"name":"suite","value":"test_default_sign_up_eps_download_welcome_modal"},{"name":"subSuite","value":"TestDefaultSignUpFlow"},{"name":"host","value":"9bfe8b625088"},{"name":"thread","value":"25-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.tests_default_flow.test_default_sign_up_eps_download_welcome_modal"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"language","value":"'en'"},{"name":"qr_create_method","value":"'vcard_qr_create'"},{"name":"resolution","value":"'Default'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"8ebed90aba785007.json","parameterValues":["'en'","'vcard_qr_create'","'Default'"]}